jcommune-view/jcommune-web-view/src/main/webapp/WEB-INF/jsp/codeReviewForm.jsp

```diff
@@ -64,4 +64,16 @@
     <spring:message code="label.back"/>
   </a>
 </div>
+<script>
+    $(function() {
+        <c:choose>
+            <c:when test="${topicDraft.id != 0}">
+                window.topicDraft.init($('.keymaps-caption'), <c:out value="${topicDraft.lastSaved.millis}"/>);
```

Comment:
Also what is 'keymap-caption'? Maybe we could come up with something more reader-friendly..